app/models/comments.rb

```diff
@@ -0,0 +1,10 @@
+class Comments < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
+
+  after_save :update_comments_counter
+
+  def update_comments_counter
+    post.update(comments_counter: post.comments.count)
+  end
```

Comment:
- [ ] **[OPTIONAL]** We can make this more concise by using the [`increment!`](https://apidock.com/rails/ActiveRecord/Base/increment!) method and we can also make this method private.

  ```
  private

  def update_comments_counter
    post.increment!(:comments_counter)
  end
  ```